diff --git a/node_modules/mt940js/lib/tags.js b/node_modules/mt940js/lib/tags.js
index 5233405..52144da 100755
--- a/node_modules/mt940js/lib/tags.js
+++ b/node_modules/mt940js/lib/tags.js
@@ -71,6 +71,9 @@ class Tag {
    * @private
    */
   _parse() {
+    // const datas = "BLABLABLA-NTRF";    
+    // datas.replace('-NTRF','NTRF');
+    // console.log("---->YEOOOOOOOOSS222",datas);    
     const match = this._nextMatch();
     if (!match) { throw Error(`Cannot parse tag ${this.id}: ${this.data}`) }
     this.fields = this._extractFields(match);
@@ -171,7 +174,7 @@ class TagStatementLine extends Tag {
              + '((\\d{2})(\\d{2}))?'      // Entry date
              + '(R?[DC])([A-Z])?'         // DC indicator + funds code
              + '([0-9,]{0,16})'           // Amount
-             + '([A-Z][A-Z0-9]{3})'       // Transaction type
+             + '-([A-Z][A-Z0-9]{3})'       // Transaction type
              + '([^/\\n]{0,16})'          // Customer reference
              + '(//(.{0,16}))?'           // Bank reference
              + '(\\n(.{0,34}))?';         // Extra
